include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,266 @@
+                                                                   " evaluated to false on " +
+                                                                   detail::make_printable(c)};
+                                             }
+                                             return c;
+                                         })
+                                       | view::char_to<value_type_t<seq_type>>,    // convert to actual target alphabet
+                                         std::back_inserter(sequence));
+        }
+        else
+        {
+            detail::consume(stream_view | view::take_until(is_end));
+        }
+        //Jump over //
+        ++stream_it;
+        ++stream_it;
+        ++stream_it;
+    }
+
+    //!\copydoc SequenceFileOutputFormat::write
+    template <typename stream_type,     // constraints checked by file
+              typename seq_type,        // other constraints checked inside function
+              typename id_type,
+              typename qual_type>
+    void write(stream_type                        & stream,
+               sequence_file_output_options const & options,
+               seq_type                           && sequence,
+               id_type                            && id,
+               qual_type                          && SEQAN3_DOXYGEN_ONLY(qualities))
+    {
+        std::ranges::ostreambuf_iterator stream_it{stream};
+        size_t sequence_size = 0;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+            sequence_size = ranges::size(sequence);
+
+        // ID
+        if constexpr (detail::decays_to_ignore_v<id_type>)
+        {
+            throw std::logic_error{"The ID field may not be set to ignore when writing genbank files."};
+        }
+        else if (ranges::empty(id)) //[[unlikely]]
+        {
+            throw std::runtime_error{"The ID field may not be empty when writing genbank files."};
+        }
+        else if (options.embl_genbank_complete_header)
+        {
+            std::ranges::copy(id, stream_it);
+        }
+        else
+        {
+            std::ranges::copy(std::string_view{"LOCUS       "}, stream_it);
+            std::ranges::copy(id, stream_it);
+            std::ranges::copy(std::string_view{"                 "}, stream_it);
+            std::ranges::copy(std::to_string(sequence_size), stream_it);
+            std::ranges::copy(std::string_view{" bp\n"}, stream_it);
+        }
+
+        // Sequence
+        if constexpr (detail::decays_to_ignore_v<seq_type>) // sequence
+        {
+            throw std::logic_error{"The SEQ field may not be set to ignore when writing genbank files."};
+        }
+        else if (ranges::empty(sequence)) //[[unlikely]]
+        {
+            throw std::runtime_error{"The SEQ field may not be empty when writing genbank files."};
+        }
+        else
+        {
+            std::ranges::copy(std::string_view{"ORIGIN"}, stream_it);
```

Comment:
```suggestion
            std::ranges::copy(std::string_view{"ORIGIN\n"}, stream_it);
```
and remove the stream_it = '\n' below